composer.json

```
@@ -20,7 +20,7 @@
         "illuminate/contracts": "^11.0 || ^12.0",
         "propaganistas/laravel-phone": "^5.3.4",
         "spatie/laravel-package-tools": "^1.19",
-        "twilio/sdk": "^7.16 || ^8.4"
+        "twilio/sdk": "^7.16 || ^8.5"
```

As you mentioned, looks like we'll need to drop support for Twilio 7 (which is fine). 👍🏻 Anyone using it can stay on an older release. 👍🏻
```suggestion
        "twilio/sdk": "^8.5"
```